src/main.rs

```diff
@@ -25,14 +28,16 @@ extern crate tracing;
 #[macro_use]
 extern crate diesel;
 
-use std::sync::{Arc, LazyLock, RwLock};
+use std::sync::{Arc, LazyLock};
```

Comment:
Use parking_lot, it will make everything nicer since they can never panic

Reply:
they are also faster and smaller in memory (but require that the platform you are deploying to can park threads)